htdocs/product/card.php

```diff
@@ -2539,38 +2539,41 @@
  * Action bar
  */
 if ($action != 'create' && $action != 'edit') {
+	$cloneProductUrl = $_SERVER["PHP_SELF"].'?action=clone&token='.newToken();
+	$cloneButtonId = 'action-clone-no-ajax';
+
 	print "\n".'<div class="tabsAction">'."\n";
 
 	$parameters = array();
 	$reshook = $hookmanager->executeHooks('addMoreActionsButtons', $parameters, $object, $action); // Note that $action and $object may have been modified by hook
 	if (empty($reshook)) {
 		if ($usercancreate) {
 			if (!isset($object->no_button_edit) || $object->no_button_edit <> 1) {
-				print '<a class="butAction" href="'.$_SERVER["PHP_SELF"].'?action=edit&token='.newToken().'&id='.$object->id.'">'.$langs->trans("Modify").'</a>';
+				print dolGetButtonAction('', $langs->trans('Modify'), 'default', $_SERVER["PHP_SELF"].'?action=edit&token='.newToken().'&id='.$object->id, '', $user->rights->societe->creer);
```

Comment:
$user->rights->societe->creer must be replaced with $usercancreate

Reply:
replace ok